core/lib/scmi-server/sub.mk

```diff
@@ -17,7 +17,8 @@ scpfw-integ-version-pat = 0
 scpfw-integ-version = $(scpfw-integ-version-maj).$(scpfw-integ-version-min).$(scpfw-integ-version-pat)
 
 srcs-y += scmi_server.c
-incdirs-y += include
+srcs-$(CFG_SCMI_SCPFW_FROM_DT) += scmi_server_scpfw.c
+global-incdirs-y += include
```

Comment:
Why is this needed (setting `global-incdirs-y` rather than `incdirs-y`)? It doesn't seem that `scmi_agent_configuration.h> is needed outside of `core/lib/scmi-server`.